### PR TITLE
WIP - Update the addon-template to match pliny generation artifacts

### PR DESCRIPTION
**Pull request by djcp**
This updates the addon-template to more closely match what you'd get if you
generated a new pliny 0.17 project and implemented this example addon on top of
it.

### CODE REVIEW

**Comment by cdwort** [comment]
We'll want to rebase this when #9 hits master.

**Comment by djcp** [comment]
@cdwort @mikehale Can you give this a relatively thorough review? I've rebased against master and will (of course) do an interactive rebase to squash before merging and clean up my ugly commit list.

Specs pass and this is the addon-template using the latest release of pliny.

**Comment by mathias** [comment]
This LGTM!

File: config.ru [hunk 1]
```diff
@@ -1,8 +1,6 @@
-require "bundler"
-Bundler.require
-
-require "./lib/initializer"
+require_relative 'lib/application'
 
 $stdout.sync = true
+$stderr.sync = true
```
Comment (mikehale) [inline config.ru:4]: I think stderr is sync'd by default, but I like the parallelism here :)

File: lib/initializer.rb [hunk 2]
```diff
@@ -34,8 +34,7 @@ def self.require_initializers
   end
 
   def self.require!(globs)
-    globs = [globs] unless globs.is_a?(Array)
-    globs.each do |f|
+    Array(globs).each do |f|
```
Comment (mikehale) [inline lib/initializer.rb:37]: ❤️

**Comment by mikehale** [comment]
Nice!

File: config.ru (hunk 1 above)
Comment (mikehale) [inline config.ru:4]:
``` console
ruby -e "p [STDOUT.sync, STDERR.sync]"
[false, true]
```

Comment (djcp) [inline config.ru:4]: good to know!

**Pull request merged**